kernel/core/io/file.cpp

```diff
@@ -4117,6 +4144,31 @@ namespace NumeRe
         _styles.Parse(sStylesContent.c_str());
         tinyxml2::XMLElement* cellXfs = _styles.FirstChildElement()->FirstChildElement("cellXfs");
 
+        // Parse through styles.xml to get numfmts
+        // Iterate through nodes
+        tinyxml2::XMLElement *numFmts = _styles.FirstChildElement()->FirstChildElement("numfmts");
+
+        // Empty set to store numfmts ID
+        std::set<int> vDateTimeIds;
+
+        if (numFmts)
+        {
+            numtFmts = numFmts->FirstChildElement();
+
+            while (numFmts)
+            {
+                int id = numFmts->IntAttribute("numFmtId");
+                std::string formatString = numFmts->Attribute("formatCode");
+
+                // Detect datetime, if yes, push to empty set
+                if (isDateTimeFormat(formatString){
+                    vDateTimeIds.insert(id); // Insert into set

```

Comment:
Push to set if formatCode attribute is of date time type. All formatCode date times have mmm-yyy of some kind